scripts/mods/mod_settings_manager.nut

```diff
@@ -0,0 +1,99 @@
+this.mod_settings_manager <- {
```

Comment:
Should have a global function to access the setting on a particular page of a particular mod. Example:
`::getModSetting(_settingID, _modID, _pageID = null)`